service/common/src/main/java/org/apache/polaris/service/exception/IcebergExceptionMapper.java

```diff
@@ -75,6 +75,8 @@ public IcebergExceptionMapper() {}
   @Override
   public Response toResponse(RuntimeException runtimeException) {
     LOGGER.info("Handling runtimeException {}", runtimeException.getMessage());
+    LOGGER.debug("Full runtimeException", runtimeException);
```

Comment:
Would you mind doing this for 500 errors as `.info`?

Reply:
Updated!